x-pack/plugins/cloud_security_posture/common/utils/get_safe_vulnerabilities_query_filter.ts

```diff
@@ -1,15 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-import { QueryDslQueryContainer } from '@elastic/elasticsearch/lib/api/types';
-
-export const getSafeVulnerabilitiesQueryFilter = (query?: QueryDslQueryContainer) => ({
-  ...query,
-  bool: {
-    ...query?.bool,
-    filter: [...((query?.bool?.filter as []) || []), { exists: { field: 'resource.id' } }],
```

Comment:
main change is removing the last filter on `getSafeVulnerabilitiesQueryFilter` to not filter out documents with missing `resource.id` as a result `getSafeVulnerabilitiesQueryFilter` itself is not needed anymore, I cleaned up all the usage